azure-pipelines.yml

```diff
@@ -183,30 +185,29 @@ stages:
               python.version: '3.9'
         services:
           postgres: postgres
+          trino: trino
         variables:
           IMAGE_SUFFIX: $[ dependencies.make_suffix.outputs['suffix.IMAGE_SUFFIX'] ]
         steps:
+          - bash: |
+              echo "Wait for trino"
+              sleep 30
+
           - bash: |
               DOCKER_TAG="$(python.version)_$(IMAGE_SUFFIX)"
-              PULL="docker pull greatexpectations/test:${DOCKER_TAG}"
-              echo ${PULL}
-              eval ${PULL}
               mkdir coverage
               PYTEST="docker run --network=host --mount type=bind,source=${PWD}/coverage,target=/coverage "
-              PYTEST+="-e GE_CLOUD_BASE_URL=$(GE_CLOUD_BASE_URL) -e GE_CLOUD_ACCESS_TOKEN=$(GE_CLOUD_ACCESS_TOKEN) "
-              PYTEST+="-e GE_CLOUD_ORGANIZATION_ID=$(GE_CLOUD_ORGANIZATION_ID) greatexpectations/test:${DOCKER_TAG} "
-              PYTEST+="/bin/bash -c \"pytest --postgresql --spark --cloud --napoleon-docstrings --no-docker-discovery "
+              PYTEST+="greatexpectations/test:${DOCKER_TAG} "
+              PYTEST+="/bin/bash -c \"pytest --postgresql --spark --trino "
```

Comment:
I have added trino to the comprehensive tests. This makes these tests take longer to run but gives us better coverage numbers. Some questions/comments:

1. I will willing to break out trino into it's own job if that is preferred. I would prefer to do that in a followup PR but can also do it in this one.
2. I am going to file a followup ticket to see if we can combine coverage numbers from individual jobs because right now the coverage numbers are not complete. This isn't supported by default but there may be a workaround:
https://github.com/microsoft/azure-pipelines-tasks/issues/10353#issuecomment-668819166

Reply:
Thanks for the research on this Bill, be default, I'm a little resistant to changes that (potentially) increase our test runtimes, especially in cases where there is a time-saving alternative which works just as well. It doesn't have to be in this PR, but would love to see `trino` broken out into its own parallizable step.

Reply:
Done. I moved `trino` to it's own `job` and removed it from here.

Reply:
👍🏼